### PR TITLE
feat: add v2.10 release

### PR DESCRIPTION
**Pull request by amagyar-iohk**

### CODE REVIEW

File: Atala PRISM/2.10.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+#### Bug fixes
+
+- Key's id name according to the DID Peer new specs
+- Recover `JTI` field correctly. Allowing to regenerate the original `JWT` string
+
+### Wallet SDK KMM
+
+#### Breaking changes
+
+- `PeerDID` specification changes
```
Comment (mkbreuningIOHK) [inline Atala PRISM/2.10.md:100]: Can we precise the spec version? Or a link to the spec?

File: Atala PRISM/2.10.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+* [Apollo Crypto Library v1.2.10](https://github.com/input-output-hk/atala-prism-apollo/releases/tag/v1.2.10)
+* [Wallet SDK Swift v5.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/releases/tag/5.0.0)
+* [Wallet SDK TypeScript v5.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts/releases/tag/v5.0.0)
+* [Wallet SDK KMM v3.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/releases/tag/v3.0.0)
+
+### Documentation
+
+* [Documentation Portal v1.81.0](https://github.com/input-output-hk/atala-prism-docs/releases/tag/v1.81.0)
+
+## Changelog
```
Comment (mkbreuningIOHK) [inline Atala PRISM/2.10.md:24]: We should have also the changes for Mediator in the release note.

**Review by mkbreuningIOHK — CHANGES_REQUESTED** [review]

File: Atala PRISM/2.10.md (hunk 2 above)
Comment (FabioPinheiro) [reply Atala PRISM/2.10.md:24]: Those are the main points:

Support protocol discover-features 2.0
Support for websockets
Support for Live Mode in the Pickup protocol
Return 202 HTTP code (for DID Comm)
Re-enable the logging based on X-Request-Id header
Minor fixes
Use Java 11 runtime
Update libs

File: Atala PRISM/2.10.md (hunk 1 above)
Comment (amagyar-iohk) [reply Atala PRISM/2.10.md:100]: @FabioPinheiro can you confirm this is the spec we are follwing?
https://github.com/decentralized-identity/peer-did-method-spec/blob/master/spec/core.md#generating-a-didpeer2

File: Atala PRISM/2.10.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+### New features
+
+- Return HTTP Status Code `202` for `DIDComm`
+- Support protocol discover-features 2.0
+- Websockets support
+- Update timeout settings for websocket
+- Update ScalaDID to 0.1.0-M18 and support new format of DID Peer 
+
+### Bug fixes
```
Comment (FabioPinheiro) [inline Atala PRISM/2.10.md:73]: We miss the did:peer specs update

File: Atala PRISM/2.10.md (hunk 1 above)
Comment (FabioPinheiro) [reply Atala PRISM/2.10.md:128]: yes

Basically, the specification was updated to remove some ambiguities. In order for different implementations to be interoperable. But these caused breaking change in all the implementations that existed of the did Peer.

We can mention for further reading check the link https://github.com/decentralized-identity/peer-did-method-spec/pull/62

**Review by mkbreuningIOHK — APPROVED** [review]

**Comment by mkbreuningIOHK** [comment]
Once the Peer DID spec update is added, it is fine to merge. @amagyar-iohk

File: Atala PRISM/2.10.md (hunk 3 above)
Comment (amagyar-iohk) [reply Atala PRISM/2.10.md:77]: added as part of breaking change

**Review by mkbreuningIOHK — APPROVED** [review]

**Pull request merged**